LanguageFeatures/Augmentation-libraries/augmenting_constructors_A01_t02.dart

```diff
@@ -3,37 +3,102 @@
 // BSD-style license that can be found in the LICENSE file.
 
 /// @assertion It is a compile-time error if:
-/// - The signature of the constructor augmentation does not match the original
-///   constructor. It must have the same number of positional parameters, the
-///   same named parameters, and matching parameters must have the same type,
-///   optionality, and any required modifiers must match. Any initializing
-///   formals and super parameters must also be the same in both constructors.
+/// - The signature of the augmenting function does not match the signature of
+///   the augmented function.
 ///
 /// @description Checks that it is a compile-time error if the signature of the
 /// constructor augmentation does not match the original constructor. Test wrong
 /// number of optional parameters.
```

Comment:
Just to avoid the ambiguity:
```suggestion
/// number of optional positional parameters.
```

Reply:
Thanks. Updated.

Reply:
Thanks!